prisma/migrations/20230101095749_simplify_provider/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- DropForeignKey
+ALTER TABLE "HealthcareProfessional" DROP CONSTRAINT "HealthcareProfessional_contactId_fkey";
+
+-- AlterTable
+ALTER TABLE "HealthcareProfessional" ALTER COLUMN "contactId" DROP NOT NULL;
```

Comment:
question: is this just for updating our current data or is this meant to be run every time?

Reply:
should this be `DROP COLUMN` instead of alter?

Reply:
This is generated by Prisma. I'll probably squash these migrations together so take a look at the next PR to see if you're happy with it.